frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -209,12 +256,19 @@ const ClusterServiceVersionStatus: React.FC<ClusterServiceVersionStatusProps> =
 export const ClusterServiceVersionTableRow = withFallback<ClusterServiceVersionTableRowProps>(
   ({ obj, key, subscription, catalogSourceMissing, index, style }) => {
     const { displayName, provider, version } = _.get(obj, 'spec');
+    const ns = UIActions.getActiveNamespace();
+    const {
+      'olm.operatorGroup': olmOperatorGroup = '',
+      'olm.targetNamespaces': olmTargetNamespaces = '',
+      'olm.operatorNamespace': olmOperatorNamespace = '',
+    } = _.get(obj, 'metadata.annotations') || {};
```

Comment:
Prefer optional chaining over lodash `get`.
```suggestion
    } = obj.metadata?.annotations || {};
```